<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SummaryDialog</class>
 <widget class="QDialog" name="SummaryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="categoryComboBox"/>
       </item>
       <item>
        <widget class="QListWidget" name="summaryListWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="sumTitle">
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Summary:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="expenseLabel">
         <property name="styleSheet">
          <string notr="true">font: 20pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
